setup.py

```diff
@@ -1,11 +1,15 @@
 #!/usr/bin/env python
 
 from os.path import exists
+import re
 from setuptools import setup
-import unification
+
+contents = re.findall(
+    r'__version__ = \'([.0-9]+)\'',
+    open('unification/__init__.py', 'r').read())[0]
```

Comment:
Nice catch! One solution would be to use `os.path.join('unification', '__init__.py')` to get a platform-specific directory separator. I'm not very familiar with Python on Windows, but [this stackoverflow thread](http://stackoverflow.com/questions/16333569/mixed-slashes-with-os-path-join-on-windows) suggests it would work. What do you think?